### PR TITLE
feat: add setting to turn off intelligent spacebar

### PR DESCRIPTION
**Pull request by singh-sp**
#1761 Intelligent spacebar can now be turned on/off in settings.

> Note: It will remain enabled for existing users.

Preview:

https://github.com/KhalisFoundation/sttm-desktop/assets/6597802/fac72d13-9885-4a27-9616-bd39e39330e6

### CODE REVIEW

File: www/configs/user-settings.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "other-options": {
       "title": "OTHER_OPTIONS",
       "type": "switch",
-      "settings": ["limit-change-log", "statistics"]
+      "settings": ["limit-change-log", "statistics", "intelligent-spacebar"]
```
Comment (Gauravjeetsingh) [inline www/configs/user-settings.json:76]: Small change, Instead of other options, can you please move this setting to App Settings.

**Review by Gauravjeetsingh — CHANGES_REQUESTED** [review]

**Comment by Gauravjeetsingh** [comment]
Also, with multipane workspace we now have another file, which contains this code of intelligent switching. 
It's in `sttm-desktop/www/main/navigator/shabad/utils/change-verse.js`. If you can update it there too, that'd be helpful.

We might get rid of shabadContent.jsx file, in later releases, once we move on to this newer code for all workspaces.

File: www/configs/user-settings.json (hunk 1 above)
Comment (singh-sp) [reply www/configs/user-settings.json:76]: Sure

**Comment by singh-sp** [comment]
@Gauravjeetsingh bhaji, unfortunately, I won't have time (a few weeks) to make those changes. Please feel free to update my branch as desired! 🙏

**Comment by Gauravjeetsingh** [comment]
Closing this out, as we have another PR for this #1989

**Pull request closed without merging**